.github/workflows/test.yaml

```diff
@@ -31,4 +31,4 @@ jobs:
 
     - name: Test
       run: |
-        pytest deepcell_datasets --cov deepcell_datasets --pep8
+        pytest deepcell_datasets --cov deepcell_datasets
```

Comment:
Is pep8 not necessary because we are running linting in the pre-commit hooks?

Reply:
Exactly, and pytest-pep8 is actually no longer maintained and this option broke with the version upgrades, which motivated me to pull the pre-commit checks. I think it started breaking with pytest 6 and the error was `Direct construction of Pep8Item has been deprecated, please use Pep8Item.from_parent.`

I forgot to drop pytest-pep8 from the requirements and will do that now